packages/plugin-pages/src/theme-plugins/page-tree/a-page-tree-builder.ts

```diff
@@ -145,6 +145,12 @@ export abstract class APageTreeBuilder implements IPageTreeBuilder {
 		}
 		this.project.registerReflection( nodeReflection );
 		this.addNodeToProjectAsChild( nodeReflection );
+    // strip a hidden node, but *after* adding its source to the project as a child
+		if( node.title === 'HIDDEN' ){
+			return node.children ?
+				this._mapPagesToReflections( node.children, parent, childrenIO ) :
+				[];
+		}
```

Comment:
I'm genuinely not clear on whether this actually works.  Once building works again, I'll go kick it until it plays nice